### PR TITLE
feat: add custom label to experiment crd

### PR DESCRIPTION
**Pull request by kerenfi**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                              |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                   |

Related issues: #XXX , #XXX ...
Closes #XXX , #XXX ...

Further  information:

### CODE REVIEW

File: src/chaos-experiments/models/chaosExperimentsManager.js [hunk 1]
```diff
@@ -102,6 +107,20 @@ module.exports.getChaosJobExperimentsByJobId = async function (jobId, contextId)
...
+const buildExperimentResource = (kubernetesChaosConfig, jobId) => {
+    const labels = {
+        [CHAOS_EXPERIMENT_LABELS.APP]: PREDATOR_RUNNER_PREFIX,
+        [CHAOS_EXPERIMENT_LABELS.JOB_ID]: jobId
+    };
+    return {
+        ...kubernetesChaosConfig,
+        metadata: {
+            ...kubernetesChaosConfig.metadata,
+            labels: labels
```
Comment (GuyAb) [inline src/chaos-experiments/models/chaosExperimentsManager.js:119]: this means that if the k8s config labels give you labels, you will delete them. Need to add to existing labels (if exist)

File: tests/unit-tests/jobs/models/jobExperimentsHandler-test.js [hunk 2]
```diff
@@ -72,8 +71,8 @@ describe('Job experiments handler tests', function () {
         clock.tick(1000);
         await jobExperimentHandler.setChaosExperimentsIfExist(jobId, jobExperiments);
         clock.tick(3000);
-        experimentsManagerRunJobStub.callCount.should.eql(2);
-        experimentsManagerRunJobStub.args[0][0].metadata.name.should.eql(`${firstExperimentName}-${experimentsManagerRunJobStub.args[0][1]}`);
+        (experimentsManagerRunJobStub.callCount).should.eql(2);
```
Comment (GuyAb) [inline tests/unit-tests/jobs/models/jobExperimentsHandler-test.js:74]: why did you add ()?

File: tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js [hunk 3]
```diff
@@ -377,10 +377,22 @@ describe('Chaos experiments manager tests', function () {
...
+                metadata: {
+                    ...chaosJobExperimentId.metadata,
+                    labels: {
+                        app: 'predator',
+                        job_id: jobId
+                    }
+                }
+            };
+
+            await manager.runChaosExperiment(mappedChaosExperiment, jobId, chaosJobExperimentId);
```
Comment (GuyAb) [inline tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js:393]: You are not supposed to call the manager with the mapped chaos experiment - should be the input k8s config and the manager should map it.

File: src/chaos-experiments/models/chaosExperimentsManager.js [hunk 4]
```diff
@@ -89,9 +93,10 @@ module.exports.insertChaosJobExperiment = async (jobExperimentId, jobId, experim
     await databaseConnector.insertChaosJobExperiment(jobExperimentId, jobId, experimentId, startTime, endTime, contextId);
 };
 
-module.exports.runChaosExperiment = async (kubernetesChaosConfig, jobExperimentId) => {
+module.exports.runChaosExperiment = async (kubernetesChaosConfig, jobId, jobExperimentId) => {
     try {
-        await connector.runChaosExperiment(kubernetesChaosConfig);
+        const mappedResource = buildExperimentResource(kubernetesChaosConfig, jobId);
```
Comment (GuyAb) [inline src/chaos-experiments/models/chaosExperimentsManager.js:98]: Can you rename to mappedKubernetesChaosConfig?

File: tests/unit-tests/jobs/models/jobExperimentsHandler-test.js (hunk 2 above)
Comment (kerenfi) [reply tests/unit-tests/jobs/models/jobExperimentsHandler-test.js:74]: correct. thanks

File: tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js (hunk 3 above)
Comment (kerenfi) [reply tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js:393]: done

File: src/chaos-experiments/models/chaosExperimentsManager.js (hunk 4 above)
Comment (kerenfi) [reply src/chaos-experiments/models/chaosExperimentsManager.js:98]: done
